activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+*   Support composite foreign keys via migration helpers.
+
+    ```ruby
+    # Assuming "carts" table has "(shop_id, id)" as a primary key.
```

Comment:
I think we are actually going to discourage using `id` as one of the ids in the composite key when we write the docs so can we change this to something like `(shop_id, oid)`?